openzwave-sys/build.rs

```diff
@@ -1,4 +1,29 @@
 extern crate gcc;
+use std::fs;
+use std::path::Path;
+use std::process::Command;
+use std::env;
+
+fn find_include_path(c: &mut gcc::Config) -> Result<(), ()> {
+    // TODO try to actually build with these paths
+    let possible_paths = vec![
+        "/usr/include/openzwave",
+        "/use/local/include/openzwave",
```

Comment:
the logic is that only if I don't find those I build using the local submodule; otherwise I use the installed lib.